src/main/java/org/sagebionetworks/web/client/widget/accessrequirements/ManagedACTAccessRequirementWidgetViewImpl.java

```diff
@@ -363,38 +361,29 @@ public void showRequestAccessModal(
     ManagedACTAccessRequirement accessRequirement,
     @Nullable RestrictableObjectDescriptor targetSubject
   ) {
-    try {
```

Comment:
Refactored so JSON serialization happens in the new utility, so the `try`/`catch` can be removed.